woocommerce/woocommerce-ios
# Store creation: update options in category/industry profiler question to match core

Part of #9397


## Description


Previously, we're showing a long grouped list of categories/industries for the store creation profiler question that roughly follow Stripe. Ghidorah team informed us about a change on the options to match the [categories in core](https://github.com/woocommerce/woocommerce/blob/07a2f83f53352bc9c24f929b7dd787c95e91f4aa/plugins/woocommerce/src/Internal/Admin/Onboarding/OnboardingIndustries.php#L28-L69) p1680838313792819-slack-C03L1NF1EA3, probably to provide a shorter list so that merchants don't have to spend time searching for the detailed industry.

This means that there is no category group anymore, and we replace the previous 2 related event properties in the Tracks event `site_creation_profiler_data`. In the free trial API request body, there's just one parameter for the industry `slug`.

Most likely, the options are going through some design changes in the next few days and I'll make a separate PR to update the options.

## Testing instructions


- Start the store creation flow either from the prologue or the store picker
- Enter a store name --> the category profiler question should be shown, with just a list of 8 options
- Tap on an option and continue --> the selling status profiler question should be shown
- Tap on any option and complete the followup question if needed --> the country profiler question should be shown
- Continue with a country --> the free trial modal should be shown
- Continue with creating a store with free trial --> a Tracks event should be logged in the console like: `🔵 Tracked site_creation_profiler_data, properties: [AnyHashable("country_code"): "AD", AnyHashable("ecommerce_platforms"): "square,wordPress", AnyHashable("is_wpcom_store"): true, AnyHashable("user_commerce_journey"): "im_already_selling_online", AnyHashable("blog_id"): 217737069, AnyHashable("industry_slug"): "electronics-computers"]`

## Screenshots


light | dark
-- | --
![Simulator Screen Shot - iPhone 14 - 2023-04-11 at 12 30 38](https://user-images.githubusercontent.com/1945542/231062601-a342e5af-9919-466e-a690-07dab129ecae.png) | ![Simulator Screen Shot - iPhone 14 - 2023-04-11 at 12 30 24](https://user-images.githubusercontent.com/1945542/231062595-9afd50a5-2599-484a-8f58-23865d98088c.png)


---
